pkg/controller/node/nodecontroller.go

```diff
@@ -505,7 +505,7 @@ func (nc *NodeController) onNodeDelete(originalObj interface{}) {
 			glog.Errorf("Received unexpected object: %v", obj)
 			return
 		}
-		node, ok = deletedState.Obj.(*v1.Node)
+		_, ok = deletedState.Obj.(*v1.Node)
```

Comment:
@wojtek-t No, in fact that node is defined in line 499. the original line 508's node is defined in the if of line 502. They are different. I passed the UT in my env.